### DISCUSSION TITLE
Mi pasteleria en progreso

### DISCUSSION THREAD

**Discussion by DaveOval** [Launch-X-Latam/MisionFrontEnd#461] [Team Posts]
Hola banda les comparto lo que llevo de la pasteleria https://github.com/DaveOval/LaunchX_Pasteleria
creo que ya esta al 100 pero me gustaria hacer una pagina que cuando le des en aceptar te muestre el resumen de tu pedido

**Comment by Strato1257** [comment]
Esta super chida, de donde bajaste la plantilla?
😁

**Comment by DaveOval** [comment]
La hice a mano utilizando como guía algunos videos y post de Instagram y tiktok jajaja si checas el código esta lleno de cosas ni le entiendo

**Comment by floressdi** [comment]
Te quedo Genial!
¿
Donde se encuentra la practica de la pastelería?
